packages/use-contractkit/src/screens/wallet-connect-custom.tsx

```diff
@@ -1,72 +1,107 @@
+import QrCode from 'qrcode.react';
```

Comment:
Since a lot changed here, would you mind adding a screenshot to this PR so I can confirm things still look right?

Reply:
You can access a deployed version of this PR in the vercel deployment here: https://use-contractkit-aemgs6fbu-c-labs.vercel.app/